src/common/models/Projection.ts

```diff
@@ -222,6 +224,9 @@ export class Projection {
     this.smoothedDailyDeaths = this.smoothWithRollingAverage(
       this.rawDailyDeaths,
     );
+    this.weeklyDeathsPer100k = this.smoothedDailyDeaths.map(
+      row => row && row * 7,
+    );
```

Comment:
I need to dive into why this is returning whole numbers instead of float. What I tried:
`... * 7`: Returns whole number
`... * 7.0`: Returns whole number
`... * 7.5`: Returns float
Without multiplying it at all: Returns float

Reply:
I think it is just the fact that deaths have to be a whole number.  You can't have 12.5 people dying in a week.

Average daily deaths is different, since it's an average # of deaths.

Reply:
Yea, but since this weekly number is inferred from an average number, shouldn't we keep the decimal?

Reply:
I guess maybe I'm not sure what you mean.  Where are you seeing us discard a decimal?

To be clear, the average daily deaths is calculated by `(sum of 7 days of deaths)/7` and we are calculating the weekly number by multiplying that by 7 which just gives us `sum of 7 days of deaths` which will always be a whole number.

Reply:
Right.  That 9.9 is probably really 9.857142(857142...). i.e. 9 and 6/7.  When you multiply that by 7 you get 69.0.

Reply:
Ah I see, that makes sense.